src/lib/parse_release.c

```diff
@@ -289,6 +289,19 @@ void parse_osinfo_for_bz(map_string_t *osinfo, char** product, char** version)
     *version = NULL;
 }
 
+void parse_osinfo_for_bug_url(map_string_t *osinfo, char** url)
+{
+    const char *os_bug_report_url = get_map_string_item_or_NULL(osinfo, "BUG_REPORT_URL");
+
+    if (os_bug_report_url)
+        *url = xstrdup(os_bug_report_url);
+    else
+        /* unset or something bad happend */
+        *url = NULL;
+
+    return;
+}
+
```

Comment:
Do we really need this function? What is the benefit? It reads just one key from a single map and there is no magic hidden there.